contracts/bridge/BridgeTransferCommon.sol

```diff
@@ -127,11 +127,16 @@ contract BridgeTransfer is BridgeHandledRequests, BridgeFee, BridgeOperator {
         if (_requestedNonce > upperHandleNonce) {
             upperHandleNonce = _requestedNonce;
         }
-        for (i = lowerHandleNonce; i <= upperHandleNonce && handleNoncesToBlockNums[i] > 0; i++) { }
-        lowerHandleNonce = i;
-        if (i != 0) {
-            recoveryBlockNumber = handleNoncesToBlockNums[i-1];
+        for (i = lowerHandleNonce; i <= upperHandleNonce && handleNoncesToBlockNums[i] > 0; i++) {
+            recoveryBlockNumber = handleNoncesToBlockNums[i];
+            delete handleNoncesToBlockNums[i];
+            delete closedValueTransferVotes[i];
```

Comment:
Since `closedValueTransferVotes` is related the multi operator, how about moving this code also?

Reply:
I think this `closedValueTransferVotes` can prevent duplicated voting. So I don't have no idea about another position.